src/components/app-header.sass

```diff
@@ -16,19 +16,30 @@ $member-size: 18px
   font-size: 14px
 
   .left
-    flex-grow: 0
+    flex: 2
     padding: 0 $padding
     display: flex
     align-items: center
-
+    justify-content: flex-start
     .problem
-      font-weight: bold
-    .class
+      text-align: center
+      color: $sage-dark-5
+      .unit
+        font-weight: bold
+      .investigation
+        font-weight: normal
+    .spacer
+      border-right: 2px solid $sage-light-1
+      height: 40px
+      margin: 5px
+    .problem-dropdown
```

Comment:
Given that the `app-header` is nominally application-common, I suspect this is going to have to be tweaked to work for Dataflow as well, perhaps by moving some of this SASS to `clue.sass`. At this point, it may just make sense to deal with that when the merge happens.